### PR TITLE
Form DPI Scaling will keep scaling when displays change over time

### LINKED ISSUE #103: DPI Scaling will keep scaling when displays change over time.
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature/enhancement request or a discussion topic

----------------------------------------

### Bug description
Describe here the issue that you are experiencing.

### Steps to reproduce the bug
- using hyphens as bullet points
- list the steps
- that reproduce the bug

**Actual result:** Describe here what happens after you run the steps above (i.e. the buggy behavior)

**Expected result:** Describe here what should happen after you run the steps above (i.e. what would be the correct behavior)
DPI scaling should only scale when the actual dpi changes. Not when the display is inactive or whatnot (mostly speaking of laptops).

### Version info

**Shotr Version:** 0.0.0

### PR DESCRIPTION
**Pull request by zacbre**
Fixes #103

### CODE REVIEW

**Comment by zacbre** [comment]
After implementing the algorithm, we have new DPI scaling changes.

I started at base 1.25x scaling.

Here's what those look like:

# 1.25x Scaling
#### Old:
1 25x-start-oldscale
#### New:
1 25x-start-newscale

# 1.5x Scaling
#### Old:
1 5x-oldscale
#### New:
1 5x-newscale

# 2x Scaling
#### Old:
2x-oldscale
#### New:
2x-newscale

File: src/Shotr.Ui/Program.cs [hunk 1]
```diff
@@ -354,35 +357,27 @@ static void CurrentDomain_UnhandledException(object sender, UnhandledExceptionEv
...
+            Application.SetHighDpiMode(HighDpiMode.PerMonitor);
         }
 
-        [DllImport("user32.dll", SetLastError = true)]
-        static extern bool SetProcessDPIAware();
-
-        public static float MeanDpIprimary = 96f;
-
-        internal static void TryEnableDpiAware()
+        internal static void CheckForCompatLayer()
```
Comment (zacbre) [inline src/Shotr.Ui/Program.cs:366]: This function enables the windows "Application Scaling" mode.

**Comment by zacbre** [comment]
Pre DPI Scaling (150%):
image


Post DPI Scaling (150%):
image

The measurement text size and the button text size have been normalized.

**Comment by zacbre** [comment]
Pre Scaled Menu:
image

Post Scaled Menu:
image

Pre Scaled Color Picker:
image

Post Scaled Color Picker:
image

**Comment by zacbre** [comment]
Updates:

Pre Scaled Region Capture Form:
image


Post Scaled Region Capture Form:
image


Pre Scaled Screen Record Form:
image


Post Scaled Screen Record Form:
image

**Comment by zacbre** [comment]
@Tursko you could test the latest changes? 🙂 thank you!

**Comment by Tursko** [comment]
One small issue found. 
Main display is 1080p no scaling. Second display is 2k 150% scaling. 

Launched Shotr on main, moved to secondary, and back to main. The logo doesn't resize correctly. Otherwise I didn't notice any issues. 
image

**Comment by zacbre** [comment]
I'm going to close this PR until this has been further worked out.

**Pull request closed without merging**